install/antennas.csv

```diff
@@ -379,6 +379,7 @@ Trimble Navigation Ltd.,TRM115000.00,1441025884,PYGR,0.002,0,0,0,2019-10-03T01:2
 Trimble Navigation Ltd.,TRM115000.00,1441025886,HORN,0.055,0,0,0,2019-09-18T01:42:59Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM115000.00,1441025890,WHNG,0.055,0,0,0,2019-09-23T05:15:16Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM115000.00,1441025913,WRPA,0.002,0,0,0,2019-09-03T03:50:30Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM115000.00,1441052890,DNVK,0.055,0,0,0,2019-10-14T03:04:44Z,9999-01-01T00:00:00Z
```

Comment:
All is well with this line, but where is the closed antenna installation for DNVK?

Reply:
I can see it and is at line 251...?